tools/cldr-code/src/main/java/org/unicode/cldr/util/UnitConverter.java

```diff
@@ -788,11 +824,15 @@ public ConversionInfo parseUnitId(
             }
         }
         metricUnit.value = outputUnit.toString();
-        return new ConversionInfo(numerator.divide(denominator), offset);
+        return new ConversionInfo(
+                numerator.divide(denominator),
+                Rational.ONE,
+                offset); // TODO: handle exponent here - how?
     }
 
     /** Only for use for simple base unit comparison */
     private class UnitComparator implements Comparator<String> {
+        // TODO: handle exponent here - how?
```

Comment:
Hmm, a comparator with an exponent factor? The only way to actually compare is to have the specific quantities; they're not comparable in the general sense. For the purposes of this impl, I think it's probably best to consider exponents as sorted before all other factors (so Ax^B sorts first by B and then by A). This gives a total ordering which is true for sufficiently large values (the limit as x approaches infinity).

Reply:
@sffc But B might be < 1. Should we check that first and then decide priority of A vs B based on that?

Reply:
If B < 1 then it should sort before everything where B = 1 (all existing units).

I think this is a fair ordering because if `x` is sufficiently large, B < 1 will cause it to be at the bottom.